src/components/ui/Select/Select.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Select as UISelect } from '@faststore/ui'
+import type { SelectProps } from '@faststore/ui'
+import { CaretDown as CaretDownIcon } from 'phosphor-react'
+
+import './select.scss'
+
+type SelectOptions = {
+  [key: string]: string
+}
+
+interface UISelectProps extends SelectProps {
+  /*
+   * Redefines the id property to be required when using the Select component. The
+   * id will be used to link the UISelect component and its label.
+   */
+  id: string
+  /*
+   * Defines the CSS class string that will be forwarded to the wrapping div "className" prop.
+   */
+  classes?: string
+  /*
+   * Defines the options available in the select. The SelectOptions object
+   * keys are the property names, while the values correspond to the text that
+   * will be displayed in the UI
+   */
+  options: SelectOptions
+  /*
+   * Specifies the text that will be displayed in the label right next to the Select.
+   * If omitted, the label will not be rendered.
+   */
+  labelText?: string
```

Comment:
What do you think about naming this variable as label?!

Reply:
Hmm I'd rather stick with `labelText` to make it explicit in the name that is only the text of the label, not the HTML label itself. To me it makes explicit that the only customizable thing here is the text inside the label.

Reply:
Yeah, you're right. good point